system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -6044,7 +6070,15 @@ class CLdapClient : implements ILdapClient, public CInterface
             employeeID_values
         };
 
-        LDAPMod *attrs[9];
+        char* employeeNumber_values[] = {(char*)employeeNumber, NULL};
```

Comment:
Check if isEmptyString(employeeNumber)?

Reply:
Have to declare the struct whether  or not employeeNumber us empty (struct is referenced below)